views/homepage/index.html

```diff
@@ -0,0 +1,153 @@
+              <div class="or-wrapper">
+                <hr />
+                <div class="or">or</div>
+              </div>
+              <div class="teach">
+                <h3>Use it to teach</h3>
+                <ul>
+                  <li>Editing HTML content</li>
+                  <li>CSS colors</li>
+                  <li>Creating an interactive element<br/> with JavaScript</li>
+                </ul>
+                <a href="{{ backToSchoolPublishedUrl }}" title="Read the Back-to-School Postcard teaching kit" class="teaching-kit">Postcard Teaching Kit</a>
+              </div>
+            </div>
+
+            <!-- Comic Strip Project -->
+            <div class="project comic-strip">
+              <h2 class="title">Current Events Comic Strip</h2>
+              <div class="thumbnail">
+                <a href="{{ comicStripRemixUrl }}" title="Remix the Current Events Comic Strip project" class="image"></a>
+              </div>
+              <p class="description">
+                Create your very own version of this comic strip.
+              </p>
+              <a href="{{ comicStripRemixUrl }}" title="Remix the Comic Strip project"  class="remix-button">Remix this Project</a>
+              <div class="or-wrapper">
+                <hr />
+                <div class="or">or</div>
+              </div>
+              <div class="teach">
+                <h3>Use it to teach</h3>
+                <ul>
+                  <li>Editing HTML content</li>
+                  <li>Finding Creative Commons-licensed images</li>
+                  <li>CSS colors</li>
+                </ul>
+                <a href="{{ comicStripPublishedUrl }}" title="Read the Comic Strip project teaching kit" class="teaching-kit">Comic Strip Teaching Kit</a>
+              </div>
+            </div>
+
+          </div>
+      </div>
+    </section>
+
+    <footer>
+      <div class="content-wrapper">
+        <div class="project-info">
+          <a class="mozilla-logo" title="mozilla" href="https://www.mozilla.org/en-US/"><img alt="mozilla" src="img/mozilla-logo.svg" /></a>
+          <p class="thimble-context">
+            Thimble is part of the Mozilla Learning Network. <br/>
+            Learn to teach like mozilla at <a href="http://teach.mozilla.org">teach.mozilla.org</a>.
+          </p>
+        </div>
+
+        <ul class="social-links">
+          <li><a title="Stuck? Get help!" class="help" href="https://support.mozilla.org/en-US/products/webmaker/thimble"><span class="icon"></span>Stuck? Get help!</a></li>
+          <li><a title="Talk to us on Twitter" class="twitter" href="https://twitter.com/mozteach"><span class="icon"></span>Talk to us on Twitter</a></li>
+          <li><a title="Contribute on Github" class="github" href="https://github.com/mozilla/thimble.webmaker.org"><span class="icon"></span>Contribute on Github</a></li>
+        </ul>
+      </div>
+    </footer>
+  </body>
```

Comment:
Let's do an experiment with load optimization.  By far the largest thing we need to load is https://mozillathimblelivepreview.net/bramble/production/dist/main.js (i.e., the require bundle for Bramble).  Let's stick the following at the bottom of the homepage's `<body>`:

```html
  <script async src="{{ variableThatPointsToBrambleMain }}"></script>
</body>
```

NOTE the presence of `async` here, and that it's at the bottom of the homepage.  What I'm hoping is that we'll be able to pre-fetch the largest bit of our app into the browser's cache while the user watches the homepage video or reads the text.